tedana/tests/test_decay.py

```diff
@@ -70,8 +70,9 @@ def test_smoke_fit_decay():
     n_times = 20
     data = np.random.random((n_samples, n_echos, n_times))
     tes = np.random.random((n_echos)).tolist()
-    mask = np.random.randint(2, size=n_samples)  # generate binary mask of random 0s and 1s
-    adaptive_mask = np.random.random((n_samples))
+    mask = np.ones(n_samples, dtype=int)
+    mask[n_samples // 2:] = 0
```

Comment:
Why does the test need the first half of the mask to be 0?

Reply:
It doesn't, but the test mask and adaptive mask can't both be random because then test failures can occur randomly.

Reply:
Maybe I haven't had enough coffee, but then wouldn't you want there to be a set number of "good" voxels rather than "bad"?

Reply:
It'll be half good and half bad. The zeros are unnecessary, but they make it a real mask.

Reply:
Cool. Add a quick comment explaining that above the first test and then this looks good to me.